cmd/manager/main.go

```diff
@@ -128,9 +128,12 @@ func newRootCommand() *cobra.Command {
 			hiveNSName := utils.GetHiveNamespace()
 			log.Infof("hive namespace: %s", hiveNSName)
 
+			// Allow an env var to disable leader election, useful when testing from source with make run to eliminate the wait time.
+			leaderElection := os.Getenv("DISABLE_LEADER_ELECTION") == ""
```

Comment:
Should this check that the value of the environment variable can be interpreted as true? I would find it unintuitive if I ran the following and leader election was disabled.
```
DISABLE_LEADER_ELECTION="false" make run
```

Reply:
omg i love this idea. I do agree we should run the value of the env var through some turn-this-into-a-boolean logic.

Reply:
Yeah good point, updated.